### PR TITLE
Able to specify per package options to nuget pack

### PR DESCRIPTION
**Pull request by jonathanmorley**
This can be useful for adding the package id to the end of the source URL for example.

### CODE REVIEW

File: lib/albacore/tasks/release.rb [hunk 1]
```diff
@@ -34,7 +35,7 @@ class Release
       include ::Rake::DSL
       include ::Albacore::DSL
 
-      def initialize name = :release, opts = {}
+      def initialize name = :release, opts = {}, &block
```
Comment (jonathanmorley) [inline lib/albacore/tasks/release.rb]: Unsure whether a block is best here, or a `with_package` parameter that accepts some kind of Proc

**Comment by haf** [comment]
Nice, can you please edit the wiki to include this?

**Comment by jonathanmorley** [comment]
https://github.com/Albacore/albacore/wiki/Release

**Comment by haf** [comment]
Excellent! Thank you so much for your contribution!

**Pull request merged**